modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-the-notificationsconfiguration-by-using-the-web-console_{context}"]
+= Configuring the NotificationsConfiguration CR by using the web console
+
+You can configure the `NotificationsConfiguration` CR by using the web console.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges and are logged into the web console.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance. For more information, see xref:../argocd_instance/argo-cd-cr-component-properties#gitops-argo-cd-notification_argo-cd-cr-component-properties[Enabling notifications with an Argo CD instance].
```

Comment:
We can't use xrefs in modules, it might work but it's not something we're supposed to do. If you really want to link to this section, you'd want to create an additional resources section under the line where this module is included in the assembly and provide an xref there. Either way, this is what this list item should be without the xref:
```suggestion
* You have enabled notifications for the Argo CD instance. For more information, see "Enabling notifications with an Argo CD instance".
```

Reply:
Fixed.